packages/react-percy-snapshot-render/package.json

```diff
@@ -15,16 +15,16 @@
     "lib"
   ],
   "peerDependencies": {
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0"
+    "react": "^15.4.0 || ^16.0.0",
+    "react-dom": "^15.4.0 || ^16.0.0"
```

Comment:
This allows users to have either React 15 or React 16 in their projects without getting any peer dependency warnings.